plugin/fzf.vim

```diff
@@ -76,6 +76,26 @@ function! fzf#complete(...)
   return call('fzf#vim#complete', a:000)
 endfunction
 
+if has('nvim') && get(g:, 'fzf_nvim_statusline', 1)
+  highlight fzf1 ctermfg=162 ctermbg=238
+  highlight fzf2 ctermfg=157 ctermbg=238
+  highlight fzf3 ctermfg=252 ctermbg=238
+
+  function! s:fzf_nvim_term()
+    if get(w:, 'airline_active', 0)
+      let w:airline_disabled = 1
+    endif
+
+    setlocal nospell
+    exec "setlocal statusline=%#fzf1#%{'\\ \\ \\>\\ '}%#fzf2#%{'fz'}%#fzf3#%{'f'}"
```

Comment:
`exec` is unnecessary; `setlocal statusline=%#fzf1#%{'\ \ \>\ '}%#fzf2#%{'fz'}%#fzf3#%{'f'}` seems to work just fine.